jwql/website/apps/jwql/monitor_models/common.py

```diff
@@ -191,15 +191,15 @@ def db_for_read(self, model, **hints):
         """
         Attempts to read monitor models go to monitors db.
         """
-        if model._meta.app_label in self.route_app_labels:
+        if model._meta.db_table_comment in self.route_app_labels:
             return "monitors"
         return None
 
     def db_for_write(self, model, **hints):
         """
         Attempts to write  monitor models go to monitors db.
         """
-        if model._meta.app_label in self.route_app_labels:
+        if model._meta.db_table_comment in self.route_app_labels:
```

Comment:
If we aren’t using `db_table_comment` to align the models with their respective db, we should use the monitor_model names from a list that should be defined in `constants.py`.  This list will be accessed inside the monitor_router class.  An example of how this would be done is: 

```
from constants import monitor_models
def db_for_write(self, model, **hints):
    if model._meta.app_label == ‘jwql’:
        if model.__name__ in monitor_models:
                return ‘monitors’
    return None
```
Note: the above suggestion should be applied to all elements in the router, with the exception of allow_migrate which will use model_name  in its check as that gathered as a parameter.

Reply:
See above, with the note that the thing that allow_migrate really uses as a parameter is "database", at least for our (single-app) setup.